changelog/fix-TEC-5105-fix-rest-api-disabled-notice

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Update the logic that displays the "REST API blocked" banner to reduce false positives.
```

Comment:
Ticket id at the end for the changelog entry please. e.g. `...positives. [TEC-XXX]`